app/code/Magento/Eav/Model/TypeLocator/SimpleType.php

```diff
@@ -50,6 +50,13 @@ public function getType($attributeCode, $entityType)
         } catch (NoSuchEntityException $exception) {
             return TypeProcessor::NORMALIZED_ANY_TYPE;
         }
+
+        $arrayFrontendInputs = ['multiselect'];
+        $frontendInput = $attribute->getFrontendInput();
+        if (in_array($frontendInput, $arrayFrontendInputs)) {
```

Comment:
```suggestion
        if ($attribute->getFrontendInput() === 'multiselect') {
```

Reply:
Hi, thank you for the review.
Do you mean integration tests?

Reply:
Nope, api-functional: https://github.com/magento/magento2/tree/2.3-develop/dev/tests/api-functional/testsuite/Magento/Eav/Api